Corona-Warn-App/src/main/res/layout/include_submission_positive_other_warning.xml

```diff
@@ -1,11 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout xmlns:android="http://schemas.android.com/apk/res/android"
+<layout xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <data>
 
         <import type="de.rki.coronawarnapp.util.formatter.FormatterAccessibilityHelper" />
 
+        <variable
+            name="countryData"
+            type="java.util.List&lt;String&gt;" />
+
+        <variable
+            name="countryListTitle"
```

Comment:
"countryListTitle" property is never used. Remove?

Reply:
Yep, will remove.